pycrostates/io/ch_data.py

```diff
@@ -115,6 +115,34 @@ def get_data(self, picks=None) -> NDArray[float]:
         data = self._data.copy()
         return data[picks, :]
 
+    def pick(self, picks, exclude="bads"):
+        """Pick a subset of channels.
+
+        Parameters
+        ----------
+        %(picks_all)s
+        exclude : list | str
+            Set of channels to exclude, only used when picking based on
+            types (e.g., exclude="bads" when picks="meg").
+        %(verbose)s
+
+        Returns
+        -------
+        inst : `~pycrostates.io.ChData`
+            The modified instance.
+        """
+        from .meas_info import ChInfo
+
+        picks = _picks_to_idx(self._info, picks, exclude=exclude)
+        data = self._data.copy()
+        data = data[picks, :]
+
+        info = pick_info(self._info, picks, copy=False)
+
+        self._data = data
+        self._info = info if isinstance(info, ChInfo) else ChInfo(info)
+        return self
```

Comment:
```suggestion
        picks = _picks_to_idx(self._info, picks, exclude=exclude)
        self._info = pick_info(self._info, picks, copy=False)
        self._data = data[picks, :]
        return self
```

Simpler: `self._info` is always a `ChInfo`, and calling `pick_info` on a `ChInfo` returns a `ChInfo`.
With the change I suggest to `_check_picks`, this method is not used anymore, but it's a nice addition.